### PR TITLE
[Homework #3] - HTML by Kamil Dróżdż

### PR DESCRIPTION
**Pull request by Kamil-Drozdz**

### CODE REVIEW

File: homeworks/kamil-drozdz/03-html/contact.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <a href="home.html">Home</a>
+            <a href="contact.html">Contact</a>
+        </nav>
+        <hr>
+    </header>
+    <main>
+        <h1>Contact Me</h1>
+        <form>
+            <label for="fullName">Full Name*</label>
+            <input type="text" id="fullName" name="fullName" required placeholder="Your name..">
```
Comment (ababenko-solvd) [inline homeworks/kamil-drozdz/03-html/contact.html:22]: Inappropriate name and id for the input, working with "firstName"

File: homeworks/kamil-drozdz/03-html/contact.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            <br>
+            <textarea id="message" name="message" rows="12" placeholder="Your message.."></textarea>
+            <br>
+            <input type="submit" value="Submit">
+        </form>
+    </main>
+    <footer>
+        <hr>
+        <div>
+            <a target="_blank" href="https://www.facebook.com"><img src="./assets/facebook.png" alt="Facebook link"
```
Comment (ababenko-solvd) [inline homeworks/kamil-drozdz/03-html/contact.html:44]: For semantic beauty and ease of reading is better to brake lines in such way:

`<a target="_blank" href="https://www.facebook.com">`
`     Facebook link`
`</a>`

File: homeworks/kamil-drozdz/03-html/home.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<html lang="pl">
```
Comment (ababenko-solvd) [inline homeworks/kamil-drozdz/03-html/home.html:1]: `<!DOCTYPE html>` is missing

File: homeworks/kamil-drozdz/03-html/home.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        <div>
+            <a href="./assets/Kamil Dróżdż Front-end Developer.pdf" download="Kamil_Dróżdż_Frontend_Developer">Download
+                Resume</a>
+        </div>
+
+    </main>
+    <footer>
+        <hr>
+        <div>
+            <a target="_blank" href="https://www.facebook.com"><img src="./assets/facebook.png" alt="Facebook link"
```
Comment (ababenko-solvd) [inline homeworks/kamil-drozdz/03-html/home.html:31]: For semantic beauty and ease of reading is better to brake lines in such way:

`<a target="_blank" href="https://www.facebook.com">`
`     Facebook link`
`</a>`

**Comment by Kamil-Drozdz** [comment]
ok great fixed now!

**Review by ababenko-solvd — APPROVED** [review]

**Review by ababenko-solvd — APPROVED** [review]

**Review by ababenko-solvd — APPROVED** [review]

**Pull request merged**